ArtDefs/Minimap.artdef

```diff
@@ -1,98 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
The deletion of this file is currently in PR in the master branch; it will not be reflected in the final version of this PR.